### PR TITLE
Update sensors.AFRICA Next.js

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #171623221

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Peek 2020-03-23 11-36

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by nerdlet** [comment]
@kilemensi The background image for `sensor stories` isn't loading 
image

**Comment by kilemensi** [comment]
> @kilemensi The background image for `sensor stories` isn't loading
> image

Yeah, seems like we were using `App.css` to style it... switched to using component `useStyles`. Should be fixed now.

**Comment by nerdlet** [comment]
Hi @karimkawambwa Not yet

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "presets": [
+    "next/babel"
+  ],
+  "plugins": [
+    [
+      "syntax-dynamic-import"
```
Comment (karimkawambwa) [inline .babelrc]: We don't need this since next js hav dynamic import `next/dynamic`

File: package.json [hunk 2]
```diff
@@ -1,77 +1,91 @@
...
+    "next": "^9.3.0",
+    "next-images": "^1.3.1",
+    "next-seo": "^4.4.0",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0",
+    "react-feather": "^2.0.3",
     "react-helmet": "^5.2.1",
```
Comment (karimkawambwa) [inline package.json]: We don't have need helmet anymore, we got `next/head`

**Comment by chegejames** [comment]
Looks LGTM

**Pull request merged**